### ISSUE TITLE
How to integrate postprocess from THREE with enable3d (phaser version)?

### ISSUE THREAD

**Issue by Trusiak** [enable3d/enable3d#40]
Hello, i know that actually you're busy by another project, but i've got a little question:

i added POSTPROCESSING like in this tutorial: 
https://redstapler.co/threejs-postprocessing-effect-tutorial/

in three.js we've got:
` renderer.render(scene,camera)`

and i have to change this for 
`composer.render();`

but here in enable 3d, we don't have control on renderer, such as in THREE.js - i mean, we have this.third.renderer, but how can i enforce this main renderer to use 

`composer.render();`

**Comment by HexaField** [comment]
I just override _update in my scene class. This is the class enable3d uses internally. I also show how I have implemented a loading screen on a different renderer and a css renderer.

This is the function I am overriding. https://github.com/enable3d/enable3d/blob/master/packages/enable3d/src/scene3d.ts#L213

 ```
_update()
{
    const delta = this.clock.getDelta() * 1000
    const time = this.clock.getElapsedTime()

    if(this.gameState === GAME_STATES.LOADING)
    {
        this.loadingScreen.update(parseFloat(time.toFixed(3)), parseInt(delta.toString()))
    } 
    else
    {
        this.update(parseFloat(time.toFixed(3)), parseInt(delta.toString()))
        this.physics.update(delta)
        this.physics.updateDebugger()
        
        this.animationMixers.update(delta)
        // this.renderer.render(this.scene, this.camera)
        this.postProcessing.render()
        this.rendererCSS.render(this.sceneCSS, this.camera)
    }
}
```

**Comment by Trusiak** [comment]
@HexaField thank you for response

`I just override _update in my scene class.`

im sory for stupid asking, but how you update "_update" from Scene3D class? 
calling "_update" won't work, because access is internall.

**Comment by Trusiak** [comment]
hey @yandeu, did you think about implement easy access to the renderer??
it could be a major leap forward in development enable3d, because it will allow to setup scene like 
[this](https://www.youtube.com/watch?v=jmld3mIKBIo).

**Comment by yandeu** [comment]
Fixed in #55
